src/ffjord.jl

```diff
@@ -159,6 +159,7 @@ function jacobian_fn(f, x::AbstractMatrix)
     for i in 1:size(y, 1)
         Zygote.@ignore z[i, :] .+= one(eltype(x))
```

Comment:
```suggestion
        Zygote.@ignore z[i, :] .= one(eltype(x))
```

both work but this is a bit more sane.

Reply:
indeed